pkg/util/iptables/testing/fake.go

```diff
@@ -37,11 +40,17 @@ type Rule map[string]string
 
 // no-op implementation of iptables Interface
 type FakeIPTables struct {
-	Lines []byte
+	iptablesFile string
```

Comment:
This class is supposed (and is) used mostly in unit-tests. That said, having it used files (doesn't matter tmp or not) is not the best option.

We should basically leave it as it was - just return the copy of lines in Save operation.